src/sandbox/eval/index.test.js

```diff
@@ -3,223 +3,225 @@ import { clearCache } from './js';
 
 describe('eval', () => {
```

Comment:
The changes in this file look big, but it's really just indented one level (because of the new `forEach` loop) and using the loop's `jsExtension` instead of static `.js`.